router/dataplane.go

```diff
@@ -145,12 +145,21 @@ type BatchConn interface {
 	// @ ensures   err == nil ==>
 	// @ 	forall i int :: { &msgs[i] } 0 <= i && i < n ==>
 	// @ 		MsgToAbsVal(&msgs[i], ingressID) == old(MultiReadBioIO_val(place, n)[i])
+	// SIF: classification spec
+	// NOTE: The postcondition for recv specifies that received messages are low.
+	// TODO: I have decided to mark the abstraction as low for now.
+	// And also used MultiReadBioIOI_val instead of MsgToAbsVal to match what is
+	// used in permissions.
+	// @ ensures err == nil ==>
+	// @ 	forall i int :: { MutliReadBioIO_val(place, n)[i] } 0 <= i && i < n ==>
+	// @		low(old(MultiReadBioIO_val(place, n)[i]))
 	ReadBatch(msgs underlayconn.Messages /*@, ghost ingressID uint16, ghost prophecyM int, ghost place io.Place @*/) (n int, err error)
 	// @ requires  acc(addr.Mem(), _)
 	// @ requires  acc(Mem(), _)
 	// @ preserves acc(sl.Bytes(b, 0, len(b)), R10)
 	// @ ensures   err == nil ==> 0 <= n && n <= len(b)
 	// @ ensures   err != nil ==> err.ErrorMem()
+	// SIF: add to classification spec later as well (bfdSend)
```

Comment:
For these kinds of TODOs, I'd rather open an issue on VerifiedSCION's issue tracker